lib/index.js

```diff
@@ -98,3 +107,39 @@ Wootric.prototype.page = function(page) {
     cacheBuster: Math.random()
   });
 };
+
+/**
+* Convert trait key to Wootric format.
+*
+* @param {string} trait
+* @param {*} value
+*/
+
+function convertKey(key, value) {
+  if (is.date(value) && !key.endsWith('_date')) return key + '_date';
+  return key;
+}
+
+/**
+ * Convert a date to unix timestamp.
+ *
+ * @api private
+ * @param {Date} date
+ * @return {number}
+ */
+
+function convertDate(date) {
+  return Math.round(date.getTime() / 1000);
+}
+
+if (!String.prototype.endsWith) {
+  String.prototype.endsWith = function(searchString, position) {
```

Comment:
@diegoserranoa could we make sure this is at least IE 9 compatible? Looks like this method has no IE support :)

Meaning this would fail our ci tests.